### DISCUSSION TITLE
Closing stale issues and PRs automatically

### DISCUSSION THREAD

**Discussion by kentakozuka** [pipe-cd/pipecd#4647] [General]
How about installing https://github.com/marketplace/actions/close-stale-issues ?

**Comment by kentakozuka** [comment]
We can change the message when issues are closed.
e.g. "The issue was closed because it has been inactive for 60 days. Feel free to reopen it 😄 "

https://github.com/marketplace/actions/close-stale-issues#close-issue-message

**Comment by kentakozuka** [comment]
Resolved by https://github.com/pipe-cd/pipecd/pull/4653

**Comment by t-kikuc** [comment]
Seems to be solved 👍
